test/FunctionalTests/BuilderTests/BuilderFilteringWebTests.cs

```diff
@@ -1,77 +1,23 @@
 using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Server.IntegrationTesting;
-using Microsoft.AspNetCore.Testing.xunit;
+using System.Net.Http;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace EntropyTests.BuilderTests
 {
-    public class BuilderFilteringWebTests
+    public class BuilderFilteringWebTests : E2ETestBase
     {
-        private const string SiteName = "Builder.Filtering.Web";
-        private readonly ITestOutputHelper _output;
         public BuilderFilteringWebTests(ITestOutputHelper output)
+            : base(output, "Builder.Filtering.Web", 5500)
```

Comment:
I suppose it's more trouble than it's worth to try making these ports unique without doing it manually.